### PR TITLE
Add documentation for KIWI.KI smartlock

### PR DESCRIPTION
**Pull request by c7h**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#14485

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/lock.kiwi.markdown [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+date: 2018-05-16 13:37
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: kiwi.png
+ha_category: Lock
+featured: false
+
+ha_release: 0.69.1
```
Comment (frenck) [inline source/_components/lock.kiwi.markdown]: Please update this to the next major release (0.70).
New features are never released in minor releases.

File: source/_components/lock.kiwi.markdown [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+```yaml
+# Example configuration.yaml entry
+lock:
+  - platform: kiwi
+    username: mail@example.com
+    password: mySecretPassword
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/lock.kiwi.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

**Review by frenck — CHANGES_REQUESTED** [review]
@c7h I left you some feedback, could you please take a look? 👍

File: source/_components/lock.kiwi.markdown (hunk 2 above)
Comment (c7h) [reply source/_components/lock.kiwi.markdown]: done :) 👍

**Comment by c7h** [comment]
@frenck The docs are updated now

File: source/_components/lock.kiwi.markdown [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+date: 2018-05-16 13:37
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: kiwi.png
+ha_category: Lock
+featured: false
+
+ha_release: 0.70
```
Comment (MartinHjelmare) [inline source/_components/lock.kiwi.markdown]: 0.72

File: source/_components/lock.kiwi.markdown [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+description: "Instructions on how to integrate KIWI Smart Lock and Smart Entry."
+date: 2018-05-16 13:37
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: kiwi.png
+ha_category: Lock
+featured: false
+
```
Comment (MartinHjelmare) [inline source/_components/lock.kiwi.markdown]: Remove this blank line.

File: source/_components/lock.kiwi.markdown (hunk 3 above)
Comment (c7h) [reply source/_components/lock.kiwi.markdown]: 👍

**Review by MartinHjelmare — APPROVED** [review]
Looks good!

**Pull request merged**